### PR TITLE
nodejs-jstz 1.0.7

### PR DESCRIPTION
**Pull request by dLobatog**
Made for theforeman/foreman#3433

### CODE REVIEW

File: nodejs-jstz/nodejs-jstz.spec [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+%files
+%{nodejs_sitelib}/%{npm_name}
+%exclude %{nodejs_sitelib}/%{npm_name}/README.md
+%exclude %{nodejs_sitelib}/%{npm_name}/package.json
+%exclude %{nodejs_sitelib}/%{npm_name}/gulpfile.js
+%exclude %{nodejs_sitelib}/%{npm_name}/test.js
+
+%files doc
+%doc README.md
+%doc LICENCE
```
Comment (domcleal) [inline nodejs-jstz/nodejs-jstz.spec]: move to main %files

**Comment by dLobatog** [comment]
Rebased

File: nodejs-jstz/nodejs-jstz.spec [hunk 2]
```diff
@@ -0,0 +1,56 @@
+%global npm_name jstz
+
+Name: nodejs-%{npm_name}
+Version: 1.0.7
+Release: 1%{?dist}
+Summary: Timezone detection for JavaScript
+License: unknown
```
Comment (domcleal) [inline nodejs-jstz/nodejs-jstz.spec]: MIT

**Comment by dLobatog** [comment]
Repushed with -doc comps and license

**Comment by domcleal** [comment]
Merged as 122efd0, thanks @dLobatog.

**Pull request closed without merging**